src/theme/NavbarItem/DropdownNavbarItem.js

```diff
@@ -5,12 +5,17 @@ import { getProductNameById } from '../../utils/getProductNameById';
 
 function useDropdownLabel(props) {
   const productId = useProductId();
-
+ 
```

Comment:
Can you format all of these js files. `cmd + shit + p` -> format document with... -> Prettier - Code Formatter. 
I will add a ticket to set-up a formatting later.

Reply:
Done.